students/stachowska_agata/lesson_01_basics/01Lesson.py

```diff
@@ -0,0 +1,102 @@
+print("1.1 «Sum of three numbers»")
+a = int(input())
+b = int(input())
+c = int(input())
+print(a + b + c)
+
+print("1.2 «Area of right-angled triangle»")
+b = int(input())
+h = int(input())
+print (0.5 * h * b)
+
+print("1.3 «Hello, Harry!»")
```

Comment:
This exercise doesn't pass snakify tests. The logic is proper, just try to avoid adding additional print statements which causes test failure.

Proper solution:
name = input("Enter your name: ")
print("Hello, " + name + "!")